src/components/bs5/button/button.scss

```diff
@@ -26,14 +26,14 @@
     --#{$prefix}btn-hover-color: var(--#{$prefix}sapphire-blue);
     --#{$prefix}btn-hover-bg: transparent;
     --#{$prefix}btn-hover-border-color: var(--#{$prefix}alt-button-hover);
-    --#{$prefix}btn-active-color: var(--#{$prefix}text-dark-blue);
+    --#{$prefix}btn-active-color: var(--#{$prefix}light-text-heading);
     --#{$prefix}btn-active-bg: var(--#{$prefix}extra-light-grey);
     --#{$prefix}btn-active-border-color: var(--#{$prefix}extra-light-grey);
     --#{$prefix}btn-active-shadow: none;
     --#{$prefix}btn-disabled-bg: transparent;
     --#{$prefix}btn-disabled-color: var(--#{$prefix}dark-grey-muted);
     --#{$prefix}btn-disabled-opacity: 1;
-    --#{$prefix}btn-disabled-border-color: var(--#{$prefix}light-grey-alt);
+    --#{$prefix}btn-disabled-border-color: var(--#{$prefix}grey);
```

Comment:
@tedqgov @stvp-qld 
as per prefix example mentioned in:
https://github.com/qld-gov-au/qgds-bootstrap5/pull/200